routers/user.router.js

```diff
@@ -1,20 +1,25 @@
-const express = require('express');
-const userController = require('./user.controller');
-const authorization = require('../middlewares/auth');
+const express = require("express");
+const userController = require("./user.controller");
+const authorization = require("../middlewares/auth");
+const { userCurrent } = require("./user.controller");
 const userRouter = express.Router();
 
 userRouter.post(
-  '/sign-up',
+  "/sign-up",
   userController.validateUserObject,
-  userController.userRegister,
+  userController.userRegister
 );
 
 userRouter.post(
-  '/sign-in',
+  "/sign-in",
   userController.validateUserObject,
-  userController.userLogin,
+  userController.userLogin
 );
 
-userRouter.get('/verify/:verificationToken', userController.verifyEmail);
+userRouter.delete("/sign-out", authorization, userController.userLogout);
```

Comment:
Method should describe an action here also

Reply:
deleteToken?

Reply:
simple `logout` would be enough. If you want to include word `user` method, it should go after word `logout`. Meaning `make some action on user`